src/code/file_menus.asm

```diff
@@ -146,60 +146,66 @@ func_001_4852::
     pop  de                                       ; $4891: $D1
     ld   a, $05                                   ; $4892: $3E $05
 
-jr_001_4894::
+.drawEmptyRowLoop
```

Comment:
If the row can contain diacritics (and thus is not always empty), maybe something like `drawRowSeparator`, or `drawSpacingRow`, `drawIntersticialRow` ?